src/components/ContractEventsTableCondensed.vue

```diff
@@ -74,19 +74,19 @@
           v-else
           :class="[
             'contract-events-table-condensed__data',
-            {'contract-events-table-condensed__data--expanded': isOpened.includes(index)}]">
+            {'contract-events-table-condensed__data--expanded': isExpanded.includes(index)}]">
           <expand-button
-            :is-collapsed="!isOpened.includes(index)"
+            :is-expanded="isExpanded.includes(index)"
```

Comment:
refactor not related to feature itself. The naming was annoying me

here is the most visible. I renamed the props root name just to `expand*`
```
<expand-button
            :is-collapsed="!isOpened.includes(index)"
```